setup.py

```diff
@@ -23,6 +23,9 @@
         "trio >= 0.16.0",
         "typing_extensions >= 3.7.4",
         "mypy_extensions >= 0.4.2",
+        "async_generator",
```

Comment:
Why do you need this backport package?

Reply:
Because otherwise you have this error: `Error importing plugin "trio_typing.plugin": No module named 'async_generator'`

Reply:
See #85